src/dataworker/Dataworker.ts

```diff
@@ -800,18 +801,22 @@ export class Dataworker {
       message: "Executing slow relay leaves",
     });
 
+    let latestRootBundles = sortEventsDescending(this.clients.hubPoolClient.getValidatedRootBundles());
+    if (this.spokeRootsLookbackCount !== 0)
+      latestRootBundles = latestRootBundles.slice(0, this.spokeRootsLookbackCount);
```

Comment:
Nit: for readability, should we surround this statement with brackets?
```suggestion
    if (this.spokeRootsLookbackCount !== 0) {
      latestRootBundles = latestRootBundles.slice(0, this.spokeRootsLookbackCount);
    }
```